packages/rpcs/cfx_sendTransaction/index.js

```diff
@@ -66,6 +70,19 @@ export const main = async ({
       throw InvalidParams(`Invalid transaction ${JSON.stringify(params)}`)
     }
 
+    if (app.currentNetwork.name === CFX_MAINNET_NAME) {
+      try {
+        const gasPrice = await cfx_gasPrice({errorFallThrough: true}, [])
```

Comment:
why use `cfx_gasPrice` not `gasstation_price`
`

Reply:
cause it's used in cfx_gasPrice https://github.com/Conflux-Chain/helios/blob/12094ee9e0a054de1fb628a7f22cbaefee6a9c72/packages/rpcs/cfx_gasPrice/index.js#L48

Reply:
`gasstation_price` is in the `cfx_gasPrice` method.